libs/schemas/src/entities/quest.schemas.ts

```diff
@@ -32,6 +32,7 @@ export const QuestActionMeta = z
     participation_period: z.nativeEnum(QuestParticipationPeriod).optional(),
     participation_times_per_period: z.number().optional(),
     created_at: z.coerce.date().optional(),
+    updated_at: z.coerce.date().optional(),
```

Comment:
The migration removes the `updated_column`, but here it's added. Should it exist or no?

Reply:
this is the action meta, which can potentially be updated (not sure)... but the migration removes updated_at in the actions table, that's append only